LuaCATS/database/GameInfo.d.lua

```diff
@@ -0,0 +1,92 @@
+--- @meta
+
+--- Usage:<br>
+--- `GameInfo.Units()` returns an iterator that loops through all rows in the `Units` table<br>
```

Comment:
You can enclouse this into lua block, it'll look nicer inside ide
\`\`\`lua

\`\`\`